pkg/api/handlers/libpod/images_pull.go

```diff
@@ -113,12 +117,39 @@ func ImagesPull(w http.ResponseWriter, r *http.Request) {
 
 	enc := json.NewEncoder(w)
 	enc.SetEscapeHTML(true)
+	copyingBlob := "Copying blob"
+	copyingConfig := "Copying config"
 	for {
 		var report entities.ImagePullReport
 		select {
+		case e := <-progress:
+			switch e.Event {
+			case types.ProgressEventNewArtifact:
+				report.Status = "pulling fs layer"
```

Comment:
I prefer keeping the status empty to be closer to how the local client looks like.